src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/util/EntityParserTest.java

```diff
@@ -59,8 +66,7 @@ public void should_parse_envelop_data_with_no_scannable_items_in() throws IOExce
 
     @Test
     public void should_parse_envelop_data_with_updated_spec_fields() throws IOException {
-        InputStream inputStream = getClass().getResourceAsStream("/metafile-model-update-rpe610.json");
-        Envelope envelope = EntityParser.parseEnvelopeMetadata(inputStream);
+        Envelope envelope = getEnvelope("/metafile-model-update-rpe610.json");
```

Comment:
`metafile-model-update-rpe610.json` what does this metafile had ? rpe610 doesn't help